packages/core/src/App/Containers/WarningScamMessageModal/warning-scam-message-modal.scss

```diff
@@ -0,0 +1,167 @@
+@mixin link-container-gaps {
+    padding: 1rem 0.5rem;
+    margin-right: 0.5rem;
+}
+
+#warning_scam_message_modal {
+    overflow: auto;
+}
+
+.warning-scam-message {
+    padding: 2.4rem 0;
+
+    @include mobile {
+        height: 80vh;
+        overflow: auto;
+    }
+
+    &--content {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        &__title {
+            display: flex;
+            flex-direction: column;
+            margin: 2.6rem 0 2.4rem;
+
+            @media screen and (max-width: 768px) {
+                width: 80%;
+            }
+
+            @media screen and (max-width: 430px) {
+                width: 90%;
+            }
+        }
+
+        &__info {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            border: solid $color-grey-4;
```

Comment:
You'll need to use `Theme.scss` variables for color to cater for dark theme changes.
If it doesn't exist then create a new variable